lib/plugins/aws/customResources/index.js

```diff
@@ -1,9 +1,21 @@
 'use strict';
 
 const path = require('path');
+const BbPromise = require('bluebird');
 const _ = require('lodash');
+const fse = BbPromise.promisifyAll(require('fs-extra'));
```

Comment:
`fs-extra` has native promise support.

```suggestion
const fse = require('fs-extra');
```

Reply:
Good catch. Unfortunately we're using a super old version of `fs-extra` and updating looked quite scary so I used the promisified solution.